test/e2e/loadtest/client/prom-statefulset.yaml

```diff
@@ -0,0 +1,105 @@
+apiVersion: apps/v1
+kind: StatefulSet
```

Comment:
Do we need separate prometheus? Why not use the existing one?

Reply:
The ones in the cluster are rotated in case they are updated/config changed/..., so they are part of the cluster update.
This one is used as a self contained load test infrastructure